### PR TITLE
Remove linaro

### PR DESCRIPTION
**Pull request by vsitdikov**
Remove linaro related stuff:

* linaro-toolchain layer
* meta-optee
* unbuildable locale

Remove psplash which breaks mentor's one

### CODE REVIEW

**Comment by chombourger** [comment]
looks good to me

**Review by chombourger — APPROVED** [review]

File: meta-arago-distro/conf/layer.conf [hunk 1]
```diff
@@ -14,8 +14,6 @@ LAYERDEPENDS_meta-arago-distro = " \
     openembedded-layer \
     networking-layer \
     meta-python \
-    linaro-toolchain \
-    meta-optee \
     qt5-layer \
     browser-layer \
```
Comment (Noor-Ahsan) [inline meta-arago-distro/conf/layer.conf]: I think we should remove this as well @chombourger

**Comment by chombourger** [comment]
I however wonder if Noor would want the BBMASK to be put in a separate layer (e.g. meta-mentor-bsp). We do not seem to pull meta-browser with our current manifest

File: meta-arago-distro/conf/layer.conf (hunk 1 above)
Comment (chombourger) [reply meta-arago-distro/conf/layer.conf]: that would be fine by me

Comment (vsitdikov) [reply meta-arago-distro/conf/layer.conf]: You mean browser-layer ?

Comment (chombourger) [reply meta-arago-distro/conf/layer.conf]: yeah

**Comment by vsitdikov** [comment]
Just have done

**Pull request merged**